code/modules/mob/living/carbon/carbon.dm

```diff
@@ -213,7 +213,6 @@
 		swap_hand()
 
 /mob/living/carbon/proc/help_shake_act(mob/living/carbon/M)
-	add_attack_logs(M, src, "Shaked", ATKLOG_ALL)
```

Comment:
I suspect we inherited this from Bay---they have a proclivity for logging just about every single interaction just to make sure "dat ARRR PEE WAS JOOSTEHFEYED!"

Reply:
BWOINK!! WHY DID YOU HEADPAT THAT CIVILIAN!?

Reply:
inb4 the server runs out of storage space